jennchenn/uwexplore
# Bread 58

## Implementation description

- Found & implemented calendar component: [react-big-calendar](https://github.com/jquense/react-big-calendar) 
- Started editing the style and view to fit what we want 
- Added sample events (BREAD-12)
- Added course details modal when clicking on a class (BREAD-19)



## Steps to test

1. Click on courses and make sure the modal appears for each one, with the correct title
2. Try resizing the screen? 



## What should reviewers focus on?

- Calendar area
